module_utils/ansible_nailgun_cement.py

```diff
@@ -229,12 +233,12 @@ def update_fields(new, old, fields):
 
 
 # Common functionality to manipulate entities
-def naildown_entity_state(entity_class, entity_dict, entity, state, module):
-    changed, _ = naildown_entity(entity_class, entity_dict, entity, state, module)
+def naildown_entity_state(entity_class, entity_dict, entity, state, module, check_missing=[]):
```

Comment:
Mutable objects as default Argumnents are almost never, what you expect. Please use` None` here.

Reply:
I did it as it saves me having to check if it's iterable later, anyway updated.